Description: Fix paths to certbund_contact's SQL files
Author: Gernot Schulz <gernot@intevation.de>
Last-Update: 2016-07-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/intelmq/bots/experts/certbund_contact/README.md
+++ b/intelmq/bots/experts/certbund_contact/README.md
@@ -14,8 +14,8 @@
     su - postgres
 
     createdb --encoding=UTF8 --template=template0 contactdb
-    psql -f intelmq/bots/experts/certbund_contact/initdb.sql   contactdb
-    psql -f intelmq/bots/experts/certbund_contact/defaults.sql contactdb
+    psql -f /usr/share/intelmq/certbund_contact/initdb.sql   contactdb
+    psql -f /usr/share/intelmq/certbund_contact/defaults.sql contactdb
 ```
 
 A database user with the right to select the data in the Contact DB
